targets/TARGET_NUVOTON/TARGET_M480/TARGET_NUMAKER_PFM_M487/mbed_overrides.c

```diff
@@ -0,0 +1,85 @@
+/* mbed Microcontroller Library
+ * Copyright (c) 2015-2016 Nuvoton
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "analogin_api.h"
+
+// NOTE: Ensure mbed_sdk_init() will get called before C++ global object constructor.
```

Comment:
is this needed? I believe for mbed OS 5, we have boot unified and should be called as needed here?